src/main/java/edu/mit/dos/object/ObjectService.java

```diff
@@ -57,19 +57,26 @@ public String create(@RequestParam("handle") String handle,
         digitalObject.setUpdateDate(new Date());
         digitalObject.setMetadataSource(metadataSystem);
         digitalObject.setMetadataSourceUrl(sourceSystem);
+        digitalObject.setTitle(title);
 
         final DigitalObject persistedObjected = objectJpaRepository.save(digitalObject);
 
         logger.debug("Persisted:{}", persistedObjected.getOid());
 
-        return "ok";
+        return String.valueOf(persistedObjected.getOid());
     }
 
 
     @RequestMapping(value = "/object", method = RequestMethod.GET)
     public @ResponseBody
-    String getObject(@RequestParam("oid") String objectId) {
-        return "ok";
+    DigitalObject getObject(@RequestParam("oid") String oid) {
+        if (objectJpaRepository == null)
+            {logger.debug("Error - object repository not found");}
+        final DigitalObject retrievedDigitalObject = objectJpaRepository.findByOid(Long.valueOf(oid));
+        if (retrievedDigitalObject == null)
+            {logger.debug("Error - digital object not found");}
```

Comment:
It might be useful to have the oid in the log. Something like ("Error, digital object not found:{}, oid)